templates/js/custom-copilot-assistant-assistants-api/README.md.tpl

```diff
@@ -96,16 +96,16 @@ Before running or debugging your bot, please follow these steps to setup your ow
 
 ### Run Teams Bot locally
```

Comment:
Bot -> Agent.

Use Bot term only for Teams Bot templates, basic bot, notification bot, command bot and workflow bot.

Bot enabled AI features, including CEA, DA, agents for Teams, avoid use Bot, instead, use Agent.

Reply:
Updated, thanks.